### ISSUE TITLE
Selecting/Highlighting text

### ISSUE THREAD

**Issue by asnoba** [kenwheeler/slick#826]
Hi,

I'm not sure whether this is a feature request or a bug, but as far as I can tell selecting/highlighting text in the active slide isn't possible – even if the `draggable` option is set to `false`. I think a fix for this would be nice for accessibility reasons. Any ideas?

I've included an example here: http://codepen.io/asnoba/full/yyNOQj/

BR,
Asger

**Comment by jmcglone** [comment]
Agreed and +1 for setting `user-select: none` when `draggable` is `true`

I see this was added and then removed from milestone 1.4–any plans to add to future milestones?

**Comment by safareli** [comment]
why was this removed?

**Comment by reloaded** [comment]
@simeydotme  Any idea if this feature is going to be implemented? Why was it decided to not implement this (or not document it)?

**Comment by nanastya** [comment]
Issue still exists. I'm sure a lot of developers would really appreciate if it was solved finally. Please, kenwheeler, help us out :)

**Comment by lingtalfi** [comment]
+1

**Comment by skyyip** [comment]
First Set the option `draggable: false` ,
Then override the css:
```
.slick-slider {
  -webkit-user-select: text;
  -moz-user-select: text;
  -ms-user-select: text;
  user-select: text;
}
```

**Comment by pedroadame** [comment]
+1
